### PR TITLE
Steam Avatar, healthbar and class on team Hud. Also partial fix to ammo display being to bright

### PR DESCRIPTION
**Pull request by AdamTadeusz**

### CODE REVIEW

**Review by nullsystem — CHANGES_REQUESTED** [review]

File: mp/src/game/client/neo/ui/neo_hud_round_state.cpp [hunk 1]
```diff
@@ -148,18 +220,36 @@ void CNEOHud_RoundState::UpdateStateForNeoHudElementDraw()
...
+	}
+	else if (inSuddenDeath)
+	{
+		prefixStr = "(Sudden death)";
+	}
+	else if (inMatchPoint)
+	{
+		prefixStr = "(Match point)";
+	}
+	V_sprintf_safe(m_szStatusANSI, "%s", prefixStr);
```
[the remark this answers is not in the document]
Comment (Agiel) [reply mp/src/game/client/neo/ui/neo_hud_round_state.cpp:238]: I'd say these are different things and we might even want to display them in different locations. "Freeze" modifies the timer (in og NT it's indicated by the random text spam in the bottom left corner but I think it's good to make it more centered), while "Match point" modifies the round counter (and should be placed nearby that).

[the remark this answers is not in the document]
Comment (AdamTadeusz) [reply mp/src/game/client/neo/ui/neo_hud_round_state.cpp]: How about Freeze and Overtime are displayed _instead_ of the time, and Match Paint and Sudden death displayed under the round counter

[the remark this answers is not in the document]
Comment (AdamTadeusz) [reply mp/src/game/client/neo/ui/neo_hud_round_state.cpp]: Or could have the timer tick down to zero during freeze in a different colour like in cs go, and tick up during overtime whenever that gets implemented

[the remark this answers is not in the document]
Comment (nullsystem) [reply mp/src/game/client/neo/ui/neo_hud_round_state.cpp]: I think a timer ticking down on different color sounds good for freeze. Regarding overtime don't need to bother with that for now until it gets implemented.

**Review by nullsystem — APPROVED** [review]
LGTM, the red countdown looks good

**Review by Agiel — APPROVED** [review]
We really shouldn't be having non-code related design discussions in PR's. I'm fine with merging this for now, but it's going to need another design pass in a more appropriate forum.

**Pull request merged**